src/Chapter1.hs

```diff
@@ -619,9 +628,10 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Integral a => a -> a
+sumLast2 n = a + b where
+    a = mod n 10
+    b = mod (mod n 100) 10
```

Comment:
It looks like you want to use `div ... 10` instead of `mod ... 10`. `mod 100` and then `mod 10` still returns the last digit.

On a side note, there is a standard function as well that combines both `div` and `mod` at once: `divMod`.

It returns the pair, so you can pattern match on that and get both variabless at once:

```haskell
let (a, b) = abs n `divMod` 10
```



That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂